fedn/fedn/network/combiner/combiner.py

```diff
@@ -134,7 +134,7 @@ def __init__(self, config):
         self.server = Server(self, self.modelservice, grpc_config)
 
         # Set up round controller
-        self.control = RoundController(config['aggregator'], self.repository, self, self.modelservice)
+        self.control = RoundController(self.repository, self, self.modelservice)
```

Comment:
to avoid confusion with controller I think it should be called round_control

Reply:
Agree, it is confusing. I changed it the RoundHandler (which is a more appropriate name)